### PR TITLE
Doc: Minor Update

### PR DESCRIPTION
**Pull request by JamesIves**
I've updated the html.md document with additional data on the title and meta description tag. This entire document details every other element besides these two so I felt it was appropriate to include them. I also adjusted some of the formatting on the headers to make them more consistent. If this type of change is welcomed I'll gladly comb through the rest of the documents and attempt to improve them where I can.

### CODE REVIEW

**Comment by clarkdemvher** [comment]
Sent from Samsung tablet.


-------- Original message --------

File: src/doc/html.md [hunk 1]
```diff
@@ -30,9 +30,20 @@ attribute to `<html>` as in this example:
 <html class="no-js" lang="en">
 ```
 
+
+## Document Title
```
Comment (arthurvr) [inline src/doc/html.md]: This makes me wonder how far our documentation should go and at what point it starts to look like an intro to HTML. This looks like general knowledge which would better fit in a getting started with web development tutorial then in the H5BP docs.

On the other hand, it's important to emphasise  things like this. What does the rest think?

Comment (clarkdemvher) [inline src/doc/html.md]: Sent from Samsung tablet.


-------- Original message --------

File: dist/doc/html.md [hunk 2]
```diff
@@ -30,9 +30,20 @@ attribute to `<html>` as in this example:
...
+
+```html
+<title>HTML5 Boilerplate</title>
+```
+
+
 ### The order of the `<title>` and `<meta>` tags
 
-The order in which the `<title>` and the `<meta>` tags are specified is
+The order in which the `<title> and the `<meta>` tags are specified is
```
Comment (drublic) [inline dist/doc/html.md]: misses closing `.

File: src/doc/html.md [hunk 3]
```diff
@@ -30,9 +30,20 @@ attribute to `<html>` as in this example:
...
+
+```html
+<title>HTML5 Boilerplate</title>
+```
+
+
 ### The order of the `<title>` and `<meta>` tags
 
-The order in which the `<title>` and the `<meta>` tags are specified is
+The order in which the `<title> and the `<meta>` tags are specified is
```
Comment (drublic) [inline src/doc/html.md]: Ah yea, see above.

File: src/doc/html.md (hunk 1 above)
Comment (drublic) [inline src/doc/html.md]: I think it's a good addition. Mostly people will know it but maybe forget about the search engine results.

File: src/doc/html.md (hunk 3 above)
Comment (JamesIves) [inline src/doc/html.md]: Fixed it up.

**Comment by alrra** [comment]
Thanks @JamesIves! :purple_heart:

**Pull request closed without merging**